core/src/wallet/bitcoin/api_impl/BitcoinLikeTransactionApi.cpp

```diff
@@ -262,6 +262,13 @@ namespace ledger {
             return *this;
         }
 
+        BitcoinLikeTransactionApi &BitcoinLikeTransactionApi::removeOutput(uint64_t index) {
```

Comment:
I think it’s okay regarding your code, but it’s easy to shoot ourselves in the foot with that kind of stuff, since it invalidates by one decrement everything right in the `_outputs` vector after the erasure.